everett/manager.py

```diff
@@ -912,15 +912,24 @@ def __call__(self, key, namespace=None, default=NO_VALUE,
                         raise
                     except Exception as orig_exc:
                         exc_info = sys.exc_info()
-                        msg = (
-                            '%s: %s; namespace=%s key=%s requires a value parseable by %s' % (
-                                exc_info[0].__name__,
-                                str(exc_info[1]),
+                        msg = [
+                            '%s: %s' % (exc_info[0].__name__, str(exc_info[1])),
+                            'namespace=%s key=%s requires a value parseable by %s' % (
                                 use_namespace,
                                 key,
                                 qualname(parser)
                             )
-                        )
+                        ]
```

Comment:
I'm increasingly of the mind that using `.format()` is not great and it's better to avoid it unless you're doing templating stuff.

I can make it more verbose with `%`, too. I'll do that.